clang-tools-extra/clangd/refactor/tweaks/RawStringLiteral.cpp

```diff
@@ -76,7 +82,7 @@ bool RawStringLiteral::prepare(const Selection &Inputs) {
   if (!N)
     return false;
   Str = dyn_cast_or_null<StringLiteral>(N->ASTNode.get<Stmt>());
-  return Str &&
+  return Str && isFeatureAvailable(Inputs) &&
```

Comment:
I would move this language mode check to the very top of `prepare()` (e.g. there is no point doing the work of `commonAncestor()` if this check fails).